public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -548,6 +553,11 @@ function AdminGSVLabelView(admin, source) {
         self.modalLabelId.html(labelMetadata['label_id']);
         self.modalStreetId.html(labelMetadata['street_edge_id']);
         self.modalRegionId.html(labelMetadata['region_id']);
+        if(labelMetadata['comments'] != null) {
```

Comment:
Another thing from the style guide is to have a space between the `if` and the `(`. Think of having `()` directly after without a space as being reserved for function calls (or indexing in some languages).